packages/vuetify/src/components/VField/VField.tsx

```diff
@@ -359,8 +362,9 @@ export const VField = genericComponent<new <T>(
 
               { props.appendInnerIcon && (
                 <InputIcon
-                  key="append-icon"
+                  key={ `append-icon-${props.glow}` }
```

Comment:
Is this necessary?

Reply:
I tried a few different approaches to get reactivity to work when props.glow is toggled:

- Passing a reactive color prop to useInputIcon
- Wrapping <InputIcon> in <VDefaultsProvider>
- Using a dynamic key on <InputIcon />

I felt the key was the cleanest. Open to suggestions.